charts/queryservice/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
           volumeMounts:
```

Comment:
is the volume mount of allowlist-static missing here?

Reply:
To be clear, if I understand correctly:
- the static list is set here
- this static list is added to the dynamic list generated by the api and stored in the main allowlist